packages/guilded.js/lib/structures/channels/Channel.ts

```diff
@@ -4,11 +4,6 @@ import { Base } from "../Base";
 import type { Message } from "../Message";
 
 export class Channel extends Base {
-    /** Send a message to this channel. */
-    send(content: RESTPostChannelMessagesBody | string): Promise<Message> {
```

Comment:
why remove this? it's pretty useful

`client.messages.send(message.channelId, content)`
vs
`message.channel.send(content)`

Reply:
Hmmm, I assumed cause as of right now, we don’t construct any channel objects that we shouldn’t have those there. I can readd them, though in the future when we do have channel objects I will most likely axe message#send then.

Reply:
right... forgot we don't have a way to do that yet